libs/tunables/tunables.c

```diff
@@ -0,0 +1,68 @@
+
+#include <avr/eeprom.h>
+#include <avr/io.h>
+
+enum function_type {
+    GETTER = 0,
+    SETTER = 1,
+    MEASURE = 2
+}
+
+// A global variable that gives each board maximum of 16 tunable parameters
+uint32_t TUNABLES_MEM[16] __attribute__((section(".eeprom")))
+= { 0 };
+
+// Can't I just declare send_can in tunables_init??????????????????????????
```

Comment:
`send_can` is a global variable that must be declared in the global scope as:

```c
static volatile bool send_can = false;
```

I can explain what `static` and `volatile` mean another time, or you can look them up!

Reply:
Also though you don't need it. see next comment.